test/integration/deprecations-test.js

```diff
@@ -33,6 +33,34 @@ describe('deprecations', () => {
     expect(console.warn.callCount).to.equal(1)
   })
 
+  it('deprecated protocol option', () => {
+    GitHub({
+      protocol: 'https'
+    })
+    expect(console.warn.callCount).to.equal(1)
+  })
```

Comment:
add `host: 'deprecations-test.com'` option to all 4 tests, because just setting `protocol` is no valid use case. That will also fix the coverage